From fbb07241ef572473eea81f17386a8f96902f4a59 Mon Sep 17 00:00:00 2001
From: lyx <lyx@rock-chips.com>
Date: Sun, 19 Sep 2010 02:52:05 -0700
Subject: [PATCH] usb: hide usb_mass_storage 2 nand flash user area is
 recognise as a usb device when connect to computer, we hide it now

---
 arch/arm/mach-rk2818/devices.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 mode change 100644 => 100755 arch/arm/mach-rk2818/devices.c

diff --git a/arch/arm/mach-rk2818/devices.c b/arch/arm/mach-rk2818/devices.c
old mode 100644
new mode 100755
index d4a241da5831..5f386acc8909
--- a/arch/arm/mach-rk2818/devices.c
+++ b/arch/arm/mach-rk2818/devices.c
@@ -625,7 +625,7 @@ struct platform_device android_usb_device = {
 };
 
 static struct usb_mass_storage_platform_data mass_storage_pdata = {
-	.nluns		= 2,
+	.nluns		= 1,
 	.vendor		= "RockChip",
 	.product	= "rk2818 sdk",
 	.release	= 0x0100,
-- 
2.35.3

